Fabric.Authorization.API/RemoteServices/IdentityProviderSearch/Providers/IdPSearchProvider.cs

```diff
@@ -56,16 +58,16 @@ public async Task<FabricIdPSearchResponse> SearchAsync(IdPPrincipalSearchRequest
             httpRequestMessage.Content = new StringContent(JsonConvert.SerializeObject(request), Encoding.UTF8,
                 "application/json");
 
-            return await ProcessResponse<IdPPrincipalSearchResponse, FabricIdPSearchResponse>(httpRequestMessage);
+            return new FabricIdPSearchResponse();
```

Comment:
I think I am missing something, but where do we invoke the actual search?

Reply:
We do not. We just use the other method in this class. I'll remove this to avoid confusion.